cpp/Platform.Data.Doublets/ILinksExtensions.h

```diff
@@ -263,16 +263,16 @@ namespace Platform::Data::Doublets
             });
             return allLinks;
         }
-    //
-    //    static Interfaces::CArray<typename TStorage::LinkAddressType>auto AllIndices<typename TStorage::LinkAddressType>(TStorage& storage, Interfaces::CArray<typename TStorage::LinkAddressType> auto&& restriction)
-    //    {
-    //        auto allIndices = List<typename TStorage::LinkAddressType>();
-    //        auto filler = ListFiller<typename TStorage::LinkAddressType, typename TStorage::LinkAddressType>(allIndices, storage.Constants.Continue);
-    //        storage.Each(filler.AddFirstAndReturnConstant, restriction);
-    //        return allIndices;
-    //    }
-    //
-    //
+
+       static Interfaces::CArray<typename TStorage::LinkAddressType>auto AllIndices<typename TStorage::LinkAddressType>(TStorage& storage, Interfaces::CArray<typename TStorage::LinkAddressType> auto&& restriction)
+        {
+            auto allIndices = List<typename TStorage::LinkAddressType>();
+            auto filler = ListFiller<typename TStorage::LinkAddressType, typename TStorage::LinkAddressType>(allIndices, storage.Constants.Continue);
```

Comment:
You should check ListFiller syntax before using it: https://github.com/linksplatform/Collections/blob/8792f34db3c51837631cf01a05871db3838ee6ac/cpp/Platform.Collections/Lists/ListFiller.h#L4